mtp_api/apps/mtp_auth/serializers.py

```diff
@@ -16,8 +16,8 @@
 
 
 class UserSerializer(serializers.ModelSerializer):
-    applications = serializers.SerializerMethodField()
-    permissions = serializers.SerializerMethodField()
+    applications = serializers.SerializerMethodField(read_only=True)
+    permissions = serializers.SerializerMethodField(read_only=True)
```

Comment:
`SerializerMethodField` is read-only by nature. Or is something complaining because it is not explicitly set?